docs/cncf_conformance.md

```diff
@@ -155,10 +155,10 @@ Let's try and list this cluster's nodes:
 ```shell
 $ ./kubectl get nodes
 NAME                                          STATUS    ROLES     AGE       VERSION
-kube-node-0-kubelet.kubernetes.mesos          Ready     <none>    2m        v1.10.7
-kube-node-1-kubelet.kubernetes.mesos          Ready     <none>    2m        v1.10.7
-kube-node-2-kubelet.kubernetes.mesos          Ready     <none>    2m        v1.10.7
-kube-node-public-0-kubelet.kubernetes.mesos   Ready     <none>    1m        v1.10.7
+kube-node-0-kubelet.kubernetes.mesos          Ready     <none>    2m        v1.12.1
```

Comment:
This list of nodes isn't updated, we should see the kube-control-plane node kube-control-plane-0. We should run the code and then update the output accordingly.